docs/docfx.json

```diff
@@ -80,80 +80,49 @@
           "excel/**",
           "excel.yml",
           "excel_1_1/**",
-          "excel_1_1.yml",
           "excel_1_2/**",
-          "excel_1_2.yml",
           "excel_1_3/**",
-          "excel_1_3.yml",
           "excel_1_4/**",
-          "excel_1_4.yml",
           "excel_1_5/**",
-          "excel_1_5.yml", 
           "excel_1_6/**",
-          "excel_1_6.yml",
           "excel_1_7/**",
-          "excel_1_7.yml",
           "excel_1_8/**",
-          "excel_1_8.yml",
           "excel_1_9/**",
-          "excel_1_9.yml",
           "excel_1_10/**",
-          "excel_1_10.yml",
           "excel_1_11/**",
-          "excel_1_11.yml",
           "excel_1_12/**",
-          "excel_1_12.yml",
           "excel_1_13/**",
-          "excel_1_13.yml",
           "excel_1_14/**",
-          "excel_1_14.yml",
           "excel_online/**",
-          "excel_online.yml",
           "office/**",
           "office.yml",
```

Comment:
Any reason we're keeping these other .yml files?

Reply:
Those yml files actually exist.
![image](https://user-images.githubusercontent.com/38896772/144483676-06f55fc6-cef2-4c99-b163-a82d89ec198b.png)